src/patternfly/components/Select/examples/Select.md

```diff
@@ -196,6 +203,7 @@ The checkbox select can select multiple items using checkboxes. The number of it
 | `.pf-c-select__menu-fieldset` | `<fieldset>` |  Initiates a fieldset for the items in a checkbox select. |
 | `.pf-c-select__menu-group` | `<div>` |  Initiates a group within a select menu. |
 | `.pf-c-select__menu-group-title` | `<div>` |  Initiates a title for a group with a select menu. |
+| `.pf-m-padding` | `.pf-c-select__menu-group` |  Adds padding to group. |
```

Comment:
Change comment to something similar to this: "Modifies the select menu group to add extra padding."